packages/core/src/sass/app/_common/components/account-common.scss

```diff
@@ -859,16 +885,23 @@
     }
 
     &__footer {
+        display: flex;
+        align-items: center;
+        justify-content: flex-end;
+        bottom: 0;
+
         @include mobile-or-tablet-screen {
-            display: flex;
```

Comment:
I saw Sergi added this css. I am not sure will this effect this work or not 😬

Reply:
i added `display :flex `outside mobile-tablet-screen 
as when this was being used in the mobile/tablet view the contents of the footer (buttons) were coming on top of one another.